scripts/budo.js

```diff
@@ -12,29 +12,39 @@ function execCmd (cmd) {
   return p;
 }
 
-var app = budo('./index.js:build/aframe.js', {
+var consts = {
+  NAME: 'AFRAME',
+  ENTRY: './src/index.js',
+  DIST: 'dist/aframe.js',
+  BUILD: 'build/aframe.js',
+  WATCH: 'examples/**/*', // Additional files to watch for LiveReload
+  PORT: 9000
+};
+
+var opts = {
   debug: process.env.NODE_ENVIRONMENT !== 'production',
   verbose: true,
-  stream: process.stdout,  // log to stdout
-  port: 9000,
-  browserifyArgs: '-s AFRAME'.split(' ')
-});
-
-app
-.watch('**/*.{css,js,html}')
-.live()
-.on('watch', function (eventType, fn) {
-  if (eventType !== 'change' && eventType !== 'add') { return; }
-
-  if (path.extname(fn) === '.css') {
-    // We want to trigger a reload of the entire document, since
-    // browserify-css injects CSS into the page.
-    app.reload();
-  } else if (path.extname(fn) === '.js') {
-    app.reload(fn);
-  } else if (fn.indexOf('elements/templates') === 0) {
-    app.reload();
+  open: Boolean(process.env.OPEN),  // To enable: `OPEN=1 npm start`
+  live: Boolean(process.env.LIVE),  // To enable: `LIVE=1 npm start`
```

Comment:
hate to nitpick, but you think we could have enable Live Reload by default (and adjust the `README` accordingly):

```js
'LIVE' in process.env ? Booelan(process.env.LIVE) : true
```

and keep `OPEN` disabled by default probably.